### PR TITLE
compatibility changes for Neos v.3.3

### PR DESCRIPTION
**Pull request by Magarif**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
     "name": "cron/neos-behat",
-    "type": "typo3-flow-package",
+    "type": "neos-package",
     "description": "Neos Behat Addons",
     "require": {
-        "typo3/flow": "*"
+        "neos/flow": "*"
```
Comment (baschny) [inline composer.json]: @Magarif After this PR, we require at least flow 4.0.0, so we should mark that explicitly:

```
    "neos/flow": "^4.0.0"
```

I will add this to the PR now.

**Comment by remuslazar** [comment]
@Magarif I don't understand ef3e1a7989384abb84b033db395e35fbf33c1691. I think the code migrations will also address the rename stuff?! Why was this needed prior to the code migrations or give me more details on that so I can understand it better.

**Comment by remuslazar** [comment]
looks good to me

**Review by remuslazar — APPROVED** [review]
looks good to me

**Pull request merged**